test/rpc_test.py

```diff
@@ -1284,6 +1284,25 @@ def test_debug_info(self):
         expected.update(autograd_info)
         self.assertEqual(expected, info)
 
+    def test_sender_exceptions(self):
+        rpc.init_rpc(
+            name="worker%d" % self.rank,
+            backend=rpc.backend_registry.BackendType[
+                dist_utils.TEST_CONFIG.rpc_backend_name
+            ],
+            rank=self.rank,
+            world_size=self.world_size,
+            rpc_backend_options=self.rpc_backend_options,
+        )
+
+        if self.rank == 0:
+            time.sleep(3)
```

Comment:
is the sleep unavoidable?

Reply:
I want to make sure that the other workers have exited before the RPC on the next line runs, to simulate an exception that can occur in `enqueueSend`, when the remote side hangs up. Not sure if there's a better way to do this.